examples/MySQL-client/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(MySQL-client-example)
 hunter_add_package(MySQL-client)
 find_package(MySQL-client REQUIRED)
 
+if(MSVC) #LIBCMT conflicts with MSVCR
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
```

Comment:
> If I alter the example to use ... it compiles fine

I think that package itself should be fixed here, not an example. Everything should be plain simple on the user's side. I personally think that this code should be removed. Another alternative is to add an option `MYSQL_CLIENT_USE_DEFAULTS`, when this option is `TRUE` don't add/change any compiler flags.